### ISSUE TITLE
Calendar view doesn't show

### ISSUE THREAD

**Issue by mkubryn** [mkubryn/herd#8]
When entering the calendar tab on /upcomming, the calendar doesn't display properly. You have to manually change, for instance, month (or simply resize the page) and then it starts looking ok.

Moreover "day, week, month" selector position is wrong.

![bug](https://cloud.githubusercontent.com/assets/5312288/15294626/6526ec6a-1b8d-11e6-92eb-3d6610611ed2.png)

**Comment by Prozi** [comment]
done

**Comment by mkubryn** [comment]
yep, the original problem has been solved (but not for mobile), however the calendar still needs some customizations:
- when opening "day" view on mobile, the view height should be bigger, to possibly cover the whole day without need to scroll
- on small mobile screens some texts overlap
